pyserini/search/reranker.py

```diff
@@ -13,12 +13,12 @@
 # limitations under the License.
 
 import enum
+import importlib
+import os
+import uuid
 from sklearn.linear_model import LogisticRegression
 from sklearn.svm import SVC
 from typing import List
-from ..vectorizer import TfidfVectorizer
-import uuid
-import os
 
 
 class ClassifierType(enum.Enum):
```

Comment:
we can kill this right? we're passing the class name as cmd-line arg?

Reply:
We are using the cmd-line class name trick for vectorizer type.

But for classifiers, it's a little tricky since different models are from different modules. It's still doable, but I think it's going to introduce more lines of code and complexity.


```
from sklearn.linear_model import LogisticRegression
from sklearn.svm import SVC
```

Reply:
okay, keep then.